manifest.php

```diff
@@ -25,6 +25,7 @@
 use oat\taoLti\controller\Security;
 use oat\taoLti\scripts\install\GenerateKeys;
 use oat\taoLti\scripts\install\InstallServices;
+use oat\taoLti\scripts\install\MapLtiSectionVisibility;
```

Comment:
tao manifest version needs to be fixed: `'tao' => '>=45.14.0'` is not existent anymore.